homeassistant/components/ephember/climate.py

```diff
@@ -61,14 +61,16 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
 
 
 class EphEmberThermostat(ClimateDevice):
-    """Representation of a HeatmiserV3 thermostat."""
+    """Representation of a EphEmber thermostat."""
 
     def __init__(self, ember, zone):
         """Initialize the thermostat."""
+        from pyephember.pyephember import zone_name, zone_is_hot_water
```

Comment:
Please move 3rd party imports to the top of the module.

Reply:
Done. 

When was this policy changed? It used to be required to not import third party code on a module basis and only do it on a local basis.

Reply:
It was changed after we introduced manifest.json.